src/patternfly/components/Panel/panel.hbs

```diff
@@ -1,4 +1,7 @@
 <div class="{{pfv}}panel{{#if panel--modifier}} {{panel--modifier}}{{/if}}"
+  {{#if panel--IsPopper}}
+    style="position: absolute; top: calc(100% + 2px); z-index: var(--pf-v5-global--ZIndex--sm);"
+  {{/if}}
```

Comment:
IMO we shouldn't include this as an inline style. It's probably better as a local demo style, just so it doesn't seem like it's part of the component.